locale/manageiq.pot

```diff
@@ -51250,10 +51288,18 @@ msgstr ""
 msgid "No message will be displayed"
 msgstr ""
 
+#: ../app/javascript/components/AeInlineMethod/NamespaceSelector.jsx
+msgid "No methods available."
+msgstr ""
+
 #: ../app/views/miq_ae_class/_class_methods.html.haml
 msgid "No methods found"
 msgstr ""
 
+#: ../app/javascript/components/AeInlineMethod/index.jsx
+msgid "No methods selected."
```

Comment:
@GilbertCherrie another minor one... all of the "No ..." messages above are missing a period `.` and these two new ones have it.  Very minor but if it makes sense to keep them consistent, seems like an easy change.

Reply:
feel free to assign out... it's a very very minor

Reply:
I fixed it in this pr: https://github.com/ManageIQ/manageiq-ui-classic/pull/9242